client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -721,45 +637,18 @@ export class PlanFeatures2023Grid extends Component< PlanFeatures2023GridType >
 				! isWooExpressPlusPlan( gridPlan.planSlug )
 		);
 
-		return plansWithFeatures.map(
-			( { planSlug, features: { wpcomFeatures, jetpackFeatures } }, mapIndex ) => {
-				return (
-					<Container
-						key={ `${ planSlug }-${ mapIndex }` }
-						isTableCell={ options?.isTableCell }
-						className="plan-features-2023-grid__table-item"
-					>
-						<PlanFeatures2023GridFeatures
-							features={ wpcomFeatures }
-							planSlug={ planSlug }
-							paidDomainName={ paidDomainName }
-							wpcomFreeDomainSuggestion={ wpcomFreeDomainSuggestion }
-							hideUnavailableFeatures={ hideUnavailableFeatures }
-							selectedFeature={ selectedFeature }
-							isCustomDomainAllowedOnFreePlan={ isCustomDomainAllowedOnFreePlan }
-						/>
-						{ jetpackFeatures.length !== 0 && (
-							<div className="plan-features-2023-grid__jp-logo" key="jp-logo">
-								<Plans2023Tooltip
-									text={ translate(
-										'Security, performance and growth tools made by the WordPress experts.'
-									) }
-								>
-									<JetpackLogo size={ 16 } />
-								</Plans2023Tooltip>
-							</div>
-						) }
-						<PlanFeatures2023GridFeatures
-							features={ jetpackFeatures }
-							planSlug={ planSlug }
-							paidDomainName={ paidDomainName }
-							wpcomFreeDomainSuggestion={ wpcomFreeDomainSuggestion }
-							hideUnavailableFeatures={ hideUnavailableFeatures }
-							isCustomDomainAllowedOnFreePlan={ isCustomDomainAllowedOnFreePlan }
-						/>
-					</Container>
-				);
-			}
+		return (
+			<PlanFeaturesContainer
+				plansWithFeatures={ plansWithFeatures }
+				paidDomainName={ paidDomainName }
+				wpcomFreeDomainSuggestion={ wpcomFreeDomainSuggestion }
+				translate={ translate }
+				hideUnavailableFeatures={ hideUnavailableFeatures }
+				selectedFeature={ selectedFeature }
+				isCustomDomainAllowedOnFreePlan={ isCustomDomainAllowedOnFreePlan }
+				isTableCell={ options?.isTableCell }
+				Container={ Container }
```

Comment:
@southp How do you feel about moving `<Container />` to a separate component in the components folder and importing that in the places where it's used?

Reply:
Good point. I think that also works :) In that case, it would be better to rename it into something more specific to its usage so whoever work on calypso wouldn't bump into it and thought they've found a generic container component. `Container` is a bit too generic.

Reply:
Agree 😄